<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="inputFileListLabel">
            <property name="text">
             <string>ROOT文件列表：</string>
            </property>
            <property name="buddy">
             <cstring>inputFileListLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inputFileListLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="resultFileNameLabel">
            <property name="text">
             <string>结果文件名称：</string>
            </property>
            <property name="buddy">
             <cstring>resultFileNameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="resultFileNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="helium3Tab">
           <attribute name="title">
            <string>生成氦3参数</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QLabel" name="helium3CentralityHighLabel">
              <property name="text">
               <string>中心度上限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3CentralityHighLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="helium3RapidityHighLabel">
              <property name="text">
               <string>快度上限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3RapidityHighLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="helium3WaitingLabel">
              <property name="text">
               <string>留着待用：</string>
              </property>
              <property name="buddy">
               <cstring>helium3WaitingLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="helium3MassLineEdit"/>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="helium3CentralityLowLineEdit"/>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="helium3PtHighLineEdit"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="helium3MassLabel">
              <property name="text">
               <string>粒子质量：</string>
              </property>
              <property name="buddy">
               <cstring>helium3MassLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="helium3CentralityHighLineEdit"/>
            </item>
            <item row="1" column="5">
             <widget class="QLineEdit" name="helium3WaitingLineEdit"/>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="helium3RapidityLowLabel">
              <property name="text">
               <string>快度下限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3RapidityLowLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="helium3CentralityLowLabel">
              <property name="text">
               <string>中心度下限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3CentralityLowLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="helium3PIDLabel">
              <property name="text">
               <string>氦3(PID)：</string>
              </property>
              <property name="buddy">
               <cstring>helium3PIDLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="helium3PtHighLabel">
              <property name="text">
               <string>横动量上限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3PtHighLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLineEdit" name="helium3RapidityLowLineEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="helium3PIDLineEdit"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="helium3PtLowLabel">
              <property name="text">
               <string>横动量下限：</string>
              </property>
              <property name="buddy">
               <cstring>helium3PtLowLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLineEdit" name="helium3RapidityHighLineEdit"/>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="helium3PtLowLineEdit"/>
            </item>
            <item row="0" column="0" colspan="6">
             <widget class="QCheckBox" name="helium3CheckBox">
              <property name="text">
               <string>是否要生成Helium3粒子？</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="antiHelium3Tab">
           <attribute name="title">
            <string>生成反氦3参数</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" colspan="6">
             <widget class="QCheckBox" name="antiHelium3CheckBox">
              <property name="text">
               <string>是否要生成反Helium3粒子？</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="antiHelium3PIDLabel">
              <property name="text">
               <string>氦3(PID)：</string>
              </property>
              <property name="buddy">
               <cstring>antiHelium3PIDLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="antiHelium3PIDLineEdit"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="antiHelium3MassLabel">
              <property name="text">
               <string>粒子质量：</string>
              </property>
              <property name="buddy">
               <cstring>antiHelium3MassLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="antiHelium3MassLineEdit"/>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="antiHelium3WaitingLabel">
              <property name="text">
               <string>留着待用：</string>
              </property>
              <property name="buddy">
               <cstring>antiHelium3WatingLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLineEdit" name="antiHelium3WatingLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>中心度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_14</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_14"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>横动量下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_17</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_17"/>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>快度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_19</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLineEdit" name="lineEdit_19"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>中心度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_18</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_18"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>横动量上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_20</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_20"/>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>快度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_12</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLineEdit" name="lineEdit_12"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>生成超氚核参数</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0" colspan="6">
             <widget class="QCheckBox" name="checkBox_5">
              <property name="text">
               <string>是否要生成超氚核？</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_45">
              <property name="text">
               <string>超氚(PID)：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_44</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_44"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>粒子质量：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_43</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_43"/>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="label_47">
              <property name="text">
               <string>留着待用：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_47</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLineEdit" name="lineEdit_47"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_44">
              <property name="text">
               <string>中心度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_40</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_40"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_40">
              <property name="text">
               <string>横动量下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_39</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_39"/>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_42">
              <property name="text">
               <string>快度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_41</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLineEdit" name="lineEdit_41"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>中心度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_46</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_46"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_46">
              <property name="text">
               <string>横动量上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_45</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_45"/>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_41">
              <property name="text">
               <string>快度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_42</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLineEdit" name="lineEdit_42"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>生产反超氚核参数</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0" colspan="6">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>是否要生成反超氚核？</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>反超氚(PID)：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_36</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_36"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>粒子质量：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_38</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEdit_38"/>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>留着待用：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_30</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLineEdit" name="lineEdit_30"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>中心度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_31</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_31"/>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_38">
              <property name="text">
               <string>横动量上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_34</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="lineEdit_34"/>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>快度下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_37</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QLineEdit" name="lineEdit_37"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>中心度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_32</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_32"/>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_37">
              <property name="text">
               <string>横动量下限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_35</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLineEdit" name="lineEdit_35"/>
            </item>
            <item row="3" column="4">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>快度上限：</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_33</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QLineEdit" name="lineEdit_33"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="quitAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/tool.png</normaloff>:/images/tool.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/help.jpg</normaloff>:/images/help.jpg</iconset>
   </property>
   <property name="text">
    <string>关于Qt和GNU/Linux</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
